test/fixtures/markdown/embedded-in-markdown/input.md

```diff
@@ -59,9 +59,7 @@ For best results, you should only have one `<style>` tag per-Astro component. Th
       }
     </style>
   </head>
-  <body>
-    ...
-  </body>
+  <body> ...</body>
```

Comment:
I am aware that this is incorrect. However, it requires implementing something we don't have at all right now: the ability for certain elements to have forced break between their children (body, html, td etc works like that)